applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_explicit_dynamic_solver.py

```diff
@@ -0,0 +1,101 @@
+import KratosMultiphysics
```

Comment:
This is general to all @KratosMultiphysics/structural-mechanics, soon we will start using Doxygen, that recognizes too the python documentation, I advise to start adding description to all python scripts in order to have a documented repository